src/OrchardCore.Modules/OrchardCore.Security/Views/SecurityHeadersSettings.Edit.cshtml

```diff
@@ -0,0 +1,361 @@
+                break;
+            case PermissionsPolicyValue.GamePad:
+                name = T["Game Pad"];
+                break;
+            case PermissionsPolicyValue.Geolocation:
+                name = T["Geolocation"];
+                break;
+            case PermissionsPolicyValue.Gyroscope:
+                name = T["Gyroscope"];
+                break;
+            case PermissionsPolicyValue.LayoutAnimations:
+                name = T["Layout Animations"];
+                break;
+            case PermissionsPolicyValue.LegacyImageFormat:
+                name = T["Legacy ImageFormat"];
+                break;
+            case PermissionsPolicyValue.Magnetometer:
+                name = T["Magnetometer"];
+                break;
+            case PermissionsPolicyValue.Microphone:
+                name = T["Microphone"];
+                break;
+            case PermissionsPolicyValue.Midi:
+                name = T["Midi"];
+                break;
+            case PermissionsPolicyValue.OversizedImages:
+                name = T["Oversized Images"];
+                break;
+            case PermissionsPolicyValue.Payment:
+                name = T["Payment"];
+                break;
+            case PermissionsPolicyValue.PictureInPicture:
+                name = T["Picture In Picture"];
+                break;
+            case PermissionsPolicyValue.PublicKeyRetrieval:
+                name = T["Public Key Retrieval"];
+                break;
+            case PermissionsPolicyValue.Push:
+                name = T["Push"];
+                break;
+            case PermissionsPolicyValue.SpeakerSelection:
+                name = T["Speaker Selection"];
+                break;
+            case PermissionsPolicyValue.ScreenWakeLock:
+                name = T["Screen Wake Lock"];
+                break;
+            case PermissionsPolicyValue.SyncXhr:
+                name = T["Synchronous XML Http Request"];
+                break;
+            case PermissionsPolicyValue.UnoptimizedImages:
+                name = T["Unoptimized Images"];
+                break;
+            case PermissionsPolicyValue.UnsizedMedia:
+                name = T["Unsized Media"];
+                break;
+            case PermissionsPolicyValue.Usb:
+                name = T["USB"];
+                break;
+            case PermissionsPolicyValue.WebShare:
+                name = T["Web Share"];
+                break;
+            case PermissionsPolicyValue.WebXR:
+                name = T["Web XR"];
+                break;
+        }
+        
+        return name;
+    }
+}
+
+<p class="alert alert-warning">
+    @T["The current tenant will be reloaded when the settings are saved."]
+</p>
+
+@if (!Model.FromAdminSettings)
+{
+    <p class="alert alert-info">
+        @T["The current settings are coming from configuration file, saving the settings will affect the AdminSettings not the configuration settings."]
```

Comment:
So, here one suggestion is

    @T["The current settings are coming from configuration sources, saving the settings will affect the AdminSettings not the configuration."]